File: Newbee.BE/NOTE.Solutions/NOTE.Solutions.BLL/Contracts/Branch/Requests/BranchRequest.cs

using NOTE.Solutions.BLL.Contracts.Employee.Requests;
using NOTE.Solutions.BLL.Contracts.POS.Requests;

namespace NOTE.Solutions.BLL.Contracts.Branch.Requests;

public class BranchRequest
{
    public string Code { get; set; } = string.Empty;
    public string BuildingNumber { get; set; } = string.Empty;
    public string Street { get; set; } = string.Empty;
    public int CityId { get; set; }
}

File: Newbee.UI/src/app/core/models/branch/requests/branch-request.ts
import { PointOfSaleRequest } from "../../point-of-sale/requests/point-of-sale-request";

export interface BranchRequest {
  buildingNumber: string;
  street: string;
  code: string;
  cityId: number;
  employeeIds : number[];
  pointOfSales : PointOfSaleRequest[];
}
